### PR TITLE
Scp244MoreAPI

### PR DESCRIPTION
**Pull request by louis1706**

### CODE REVIEW

File: Exiled.API/Features/Items/Scp244.cs [hunk 1]
```diff
@@ -54,6 +54,22 @@ public bool Primed
             set => Base._primed = value;
         }
 
+        /// <summary>
+        /// Gets or sets the Scp244's remaining health.
```
Comment (Misaka-ZeroTwo) [inline Exiled.API/Features/Items/Scp244.cs:58]:
```suggestion
        /// Gets or sets the health of Scp244.
```

File: Exiled.API/Features/Items/Scp244.cs [hunk 2]
```diff
@@ -54,6 +54,22 @@ public bool Primed
...
+        /// </summary>
+        public float Health { get; set; }
+
+        /// <summary>
+        /// Gets or sets the activation angle, where 1 is a minimum, and -1 it's a maximum activation angle.
```
Comment (Misaka-ZeroTwo) [inline Exiled.API/Features/Items/Scp244.cs:63]:
```suggestion
        /// Gets or sets the activation angle, where 1 is the minimum and -1 is the maximum activation angle.
```

File: Exiled.API/Features/Items/Scp244.cs [hunk 3]
```diff
@@ -54,6 +54,22 @@ public bool Primed
...
+        /// </summary>
+        public float ActivationDot { get; set; }
+
+        /// <summary>
+        /// Gets or sets the max size of the Scp244's Hypothermia effect is dealt.
```
Comment (Misaka-ZeroTwo) [inline Exiled.API/Features/Items/Scp244.cs:68]:
```suggestion
        /// Gets or sets the maximum diameter within which SCP-244's hypothermia effect is dealt.
```

File: Exiled.API/Features/Pickups/Scp244Pickup.cs [hunk 4]
```diff
@@ -75,6 +76,16 @@ public float CurrentSizePercent
             set => Base.CurrentSizePercent = value;
         }
 
+        /// <summary>
+        /// Gets or sets the max size of the Scp244's Hypothermia effect is dealt.
```
Comment (Misaka-ZeroTwo) [inline Exiled.API/Features/Pickups/Scp244Pickup.cs:80]:
```suggestion
        /// Gets or sets the maximum diameter within which SCP-244's hypothermia effect is dealt.
```

File: Exiled.API/Features/Pickups/Scp244Pickup.cs [hunk 5]
```diff
@@ -75,6 +76,16 @@ public float CurrentSizePercent
...
+        /// </summary>
+        /// <remarks>This do not prevent visual effect.</remarks>
```
Comment (Misaka-ZeroTwo) [inline Exiled.API/Features/Pickups/Scp244Pickup.cs:82]:
```suggestion
        /// <remarks>This does not prevent visual effects.</remarks>
```

File: Exiled.API/Features/Items/Scp244.cs [hunk 6]
```diff
@@ -54,6 +54,22 @@ public bool Primed
...
+        /// </summary>
+        /// <remarks>This do not prevent visual effect.</remarks>
```
Comment (Misaka-ZeroTwo) [inline Exiled.API/Features/Items/Scp244.cs:70]:
```suggestion
        /// <remarks>This does not prevent visual effects.</remarks>
```

**Review by Misaka-ZeroTwo — COMMENTED** [review]
Grammar review

File: Exiled.API/Features/Items/Scp244.cs (hunk 1 above)
Comment (Thundermaker300) [reply Exiled.API/Features/Items/Scp244.cs:58]: This is wrong, it should be unchanged but the "the" before "Scp244's" should be removed

**Review by Misaka-ZeroTwo — APPROVED** [review]
lgtm

**Pull request merged**